src/models/searchservice/response/FieldValueDirectAnswer.ts

```diff
@@ -1,23 +1,25 @@
+import { EnumOrLiteral } from '../../utils/EnumOrLiteral';
 import { DirectAnswer } from './DirectAnswer';
 import { DirectAnswerType } from './DirectAnswerType';
+import { FieldType } from './FieldType';
 import { Result } from './Result';
 
 /**
  * A direct answer where the answer came from a field from the knowledge graph.
  *
  * @public
  */
-export interface FieldValueDirectAnswer extends DirectAnswer {
+export interface FieldValueDirectAnswer<T = unknown> extends DirectAnswer<T> {
   /** {@link DirectAnswerType}.FieldValue. */
   type: DirectAnswerType.FieldValue,
   /** {@inheritDoc DirectAnswer.value} */
-  value: string,
+  value: T,
   /** {@inheritDoc DirectAnswer.relatedResult} */
   relatedResult: Result,
   /** {@inheritDoc DirectAnswer.verticalKey} */
   verticalKey: string,
   /** {@inheritDoc DirectAnswer.fieldType} */
-  fieldType: string,
+  fieldType: EnumOrLiteral<FieldType>,
```

Comment:
Should fieldType here (and in the other places) be `fieldType: EnumOrLIteral<FieldType> | string` in order to handle custom field types, or future built in fieldTypes we don't have handling for?

Reply:
I was thinking we could even use a template literal type like
```ts
EnumOrLIteral<FieldType> | `c_${string}`
```
but it doesn't seem super useful and I don't really want to make assumptions about live API

Reply:
Decided to make a custom field with custom field type for Person entity in our test repo.
![Screen Shot 2022-08-11 at 3 23 50 PM](https://user-images.githubusercontent.com/36055303/184222469-94bb47b0-8ecb-4df7-9d01-4d3371846cdf.png)
Looks like it doesn't follow the format of c_blah, I think having EnumOrLiteral<FieldType> | string make sense. Although if we are doing | string, then having EnumOrLiteral is probably not needed.

Reply:
Decided to make a custom field with custom field type for Person entity in our test repo.

![Screen Shot 2022-08-11 at 3 30 02 PM](https://user-images.githubusercontent.com/36055303/184223600-1425c81b-bc0f-4f43-b4e3-c59425dcc710.png)

Looks like it doesn't follow the format of c_blah, I think having EnumOrLiteral<FieldType> | string make sense. Although if we are doing | string, then having EnumOrLiteral is probably not needed?
I think `FieldType | string` should be good!

Reply:
that sgtm! ... it's weird that fieldType is such an ugly string here